homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -32,9 +32,9 @@ def __init__(self, discovery_info):
         self._host = discovery_info.get("host")
         self._port = discovery_info.get("port")
         properties = discovery_info.get("properties")
-        self._uuid = properties.get("UUID")
+        self._hostname = properties.get("hostname")
```

Comment:
I tried with a SK10Y and hostname is not included in the discovery apparently.

```
File "/config/custom_components/lg_soundbar/media_player.py", line 37, in __init__
    self._name = self._hostname.split(".")[0]
AttributeError: 'NoneType' object has no attribute 'split'
```

Reply:
In fact, I see the hostname in `discovery_info` but not in `properties`

Reply:
Oh, i have some more changes in my custom_component that i am testing, somehow i messed it up. Diffed it again and fixed the commit.

Reply:
Thanks ! 👍